onnx_tf/handlers/backend/lstm.py

```diff
@@ -126,8 +126,11 @@ def _common(cls, node, **kwargs):
 
     if "clip" in node.attrs:
       cell_kwargs["cell_clip"] = node.attrs["clip"]
-
+  
     tf_activations = [tf.nn.tanh]
+    if num_directions == 2:
```

Comment:
I believe the issue https://github.com/onnx/onnx-tensorflow/issues/327 is concerning when the optional attribute "activations" is not present. If the attribute is there, the current code should be good. Maybe move the append to if "activations" in node.attrs:.... else: if num_directions == 2: tf_activations.append(tf.nn.tanh)

Reply:
see the new commit, sorry for late relpy